CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## [1.0.7] - 2022-09-13
+
+### Added
+
+- Added support for multiple extras in a single flag. [#103](https://github.com/python-poetry/poetry-plugin-export/pull/103)
+- Added `homepage` and `repository` in metadata. [#113](https://github.com/python-poetry/poetry-plugin-export/pull/113)
```

Comment:
Is this really worth mentioning? I don't think that it's interesting for the users to know about that, since it's chore stuff.